File: 0001-two-sum/0001-two-sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int k) {
        int n = nums.size();
        for(int i = 0; i < n-1 ; i++){
            for(int j = i + 1; j < n; j++){
                 int diff = nums[i] + nums[j];
                 if(diff == k){
                    return{i,j};
                 }
                }
            }
            return{};
        
    }
};

File: 0001-two-sum/0001-two-sum.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(nums, k) {
 let n = nums.length;
        for(let i = 0; i < n-1 ; i++){
            for(let j = i + 1; j < n; j++){
                 let diff = nums[i] + nums[j];
                 if(diff == k){
                    return[i,j];
                 }
                }
            }
            return[];
};